tools/project_manager/edit_common.inc

```diff
@@ -316,10 +316,12 @@ function maybe_echo_placeholder_option($initial)
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-function proj_comments_field($comments)
+function proj_comments_field($comments, $field_name, $comment_format)
 {
     $enc_comments = html_safe($comments);
 
+    $comment_format_text = $comment_format === 'html' ? _("HTML") : "<a href='https://www.pgdp.net/wiki/Markdown'>" . _("Markdown") . "</a>";
+    echo _("Comment format: ") . $comment_format_text . "<br>";
```

Comment:
We usually try to separate out label strings from formatting in case we re-use the labels elsewhere without the colon, i.e.
```php
    echo _("Comment format") . ": $comment_format_text<br>";
```